features/package.json

```diff
@@ -39,6 +39,7 @@
         "axios": "^0.19.2",
         "codemirror": "^5.52.0",
         "country-language": "^0.1.7",
+        "dayjs": "^1.11.18",
```

Comment:
Instead of adding `dayjs` to the package.json of features folder, shall we add the dependency only to the feature modules that use the dayjs package?

Reply:
Yes got it!